<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuidDISetting</class>
 <widget class="QWidget" name="GuidDISetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,9,1">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>The device shapes are divided into two types. The .DI device shown in the diagram below is optional and added according to actual requirements. If you choose to add, configure the appropriate parameters after the completion of the click on the next step, while directly click on the next step.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="radioButton_set">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_ignore">
       <property name="text">
        <string>Do not Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="vLayout_content"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="vLayout_content_crash"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_warningToolTip">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Note: the maximum number of the above two devices combined is 16</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
